### DISCUSSION TITLE
More stats information in output

### DISCUSSION THREAD

**Discussion by mrexox** [evilmartians/lefthook#339] [Ideas]
There is a suggestion to add more information to lefthook output. For example

- Execution time of every command/script in a hook
- A note which command/script took too much time to execute

You might want to suggest some more things to consider. Welcome!

**Comment by mrexox** [comment]
@abitrolly, would you like to add something else?

**Reply by abitrolly** [reply]
Memory usage and disk I/O to figure out if there is an I/O bottleneck. I remember Linux has some abstractions like `load`, but I don't remember how to read it.
